### PR TITLE
Add release documentation

### PR DESCRIPTION
**Pull request by jerome-jutteau**
Signed-off-by: Jérôme Jutteau <jerome.jutteau@outscale.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,3 +38,25 @@ We’ll do our best to review your pull request (or “PR” for short) quickly.
...
 merged - if you have ten small, unrelated changes, then go ahead and submit ten PRs.
+
+# How to release
+
+- Be sure to be rebased on `master` branch
+- Edit `osc_sdk/sdk.py` and update `SDK_VERSION` following [semantic versioning 2.0.0](https://semver.org/)
+- Update `setup.py` to update `version`
+- Run all tests with `make test`
+- Build with `make build` and check `dist` folder
+- Go to pkg and run `make`. Make sure you have `osc-cli-x86_64.zip` and `osc-cli-x86_64.AppImage`
```
Comment (outscale-mgo) [inline CONTRIBUTING.md:49]: maybe a reminder, that you need either docker or packman for that
and if my PR is accepted, we would need `./configure; make`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,3 +38,25 @@ We’ll do our best to review your pull request (or “PR” for short) quickly.
...
+- Create PR to run github actions and get validation
+- Make the PR reviewed
+- Create release page
+- Add major change to release page
+- Upload packages in release page
+- Check that auto-build workflow has published package to pip
+- Update homebrew:
+  - Update [osc-cli's brew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/osc-cli.rb#L6)
+  - Update dependencies with `brew update-python-resources osc-cli`
+  - Create PR and follow asked instructions in template
```
Comment (outscale-mgo) [inline CONTRIBUTING.md:62]: `Create PR and follow asked instructions in PR template` 
so we avoir the question "which tenplate"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -38,3 +38,25 @@ We’ll do our best to review your pull request (or “PR” for short) quickly.
...
+- Build with `make build` and check `dist` folder
+- Go to pkg and run `make`. Make sure you have `osc-cli-x86_64.zip` and `osc-cli-x86_64.AppImage`
+- Commit release update: `git commit -am "osc-cli v1.x.x"`
+- Tag new release `git tag v1.x.x`
+- Push release to new branch `git push --tags outscale master:v1.x.x`
+- Create PR to run github actions and get validation
+- Make the PR reviewed
+- Create release page
+- Add major change to release page
+- Upload packages in release page
```
Comment (outscale-mgo) [inline CONTRIBUTING.md:57]: - Upload `osc-cli-x86_64.zip` and `osc-cli-x86_64.AppImage` packages in release page

**Review by outscale-mgo — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (jerome-jutteau) [reply CONTRIBUTING.md:62]: So, which template are you using for this task?

File: CONTRIBUTING.md (hunk 3 above)
Comment (jerome-jutteau) [reply CONTRIBUTING.md:57]: Yes, better than implicit.
We can also ad `dist/osc_sdk-1.6-py2.py3-none-any.whl` and `dist/osc-sdk-1.6.tar.gz`.

**Pull request merged**